modules/InboundEmail/InboundEmail.php

```diff
@@ -548,22 +548,21 @@ function ($a, $b) use ($sortCRM) {  // defaults to DESC order
      * @param $imapStructure
      * @return bool
      */
-    private function mesageStructureHasAttachment($imapStructure)
+    public function mesageStructureHasAttachment($imapStructure)
```

Comment:
This is just a detail but the typo of `mesage` instead of `message` might be worth correcting?